src/storage/page_buffer.c

```diff
@@ -1834,7 +1834,8 @@ pgbuf_fix_release (THREAD_ENTRY * thread_p, const VPID * vpid, PAGE_FETCH_MODE f
     {
       /* Make sure that the page has been allocated (i.e., is a valid page) */
       /* Suppress errors if fetch mode is OLD_PAGE_IF_IN_BUFFER. */
-      if (pgbuf_is_valid_page (thread_p, vpid, fetch_mode == OLD_PAGE_IF_IN_BUFFER, NULL, NULL) != DISK_VALID)
+      const bool no_error_when {fetch_mode == OLD_PAGE_IF_IN_BUFFER || fetch_mode == OLD_PAGE_IF_IN_BUFFER};
```

Comment:
@ribram 
I think changes in this file are not needed anymore. I have started a test suite without these changes just to validate this assumption.

Reply:
@cristiarg me too.

Reply:
Done.